tests/unit/core/config/test_user_config.py

```diff
@@ -162,25 +173,159 @@ def test_load_ignored_matches_with_empty_names(self):
             IgnoredMatch(name="", match="dbca"),
         ]
 
-    def test_iac_config(self, local_config_path):
+    @pytest.mark.parametrize(
+        "paths",
+        (
+            ["mypath"],
+            [{"path": "mypath"}],
+            [
+                {
+                    "path": "mypath",
+                    "comment": "mycomment",
+                    "until": "2050-01-01 00:00:00",
+                }
+            ],
+            ["myfirstpath", {"path": "mysecondpath", "comment": "mycomment"}],
+        ),
+    )
+    @pytest.mark.parametrize(
+        "policies",
+        (
+            ["GG_IAC_0001"],
+            [{"policy": "GG_IAC_0001"}],
+            [
+                {
+                    "policy": "GG_IAC_0001",
+                    "comment": "mycomment",
+                    "until": "2050-01-01 00:00:00",
+                }
+            ],
+            ["GG_IAC_0001", {"policy": "GG_IAC_0002", "comment": "mycomment"}],
+        ),
+    )
```

Comment:
We have a test with `comment` but without `until`, we should also have a test with `until` but without `comment`

+ a test with duplicate entry would be great